src/Tizen.NUI/src/public/Layouting/LayoutItem.cs

```diff
@@ -73,6 +73,14 @@ public class LayoutItem
         [EditorBrowsable(EditorBrowsableState.Never)]
          public bool LayoutWithTransition{get; set;}
 
+        /// <summary>
+        /// [Draft] Set position by layouting result
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
This looks hidden API, so please remove <since_tizen> tag and the comment of ///This will be public opened ...

Reply:
DONE